sites-available/matomo.conf

```diff
@@ -53,7 +53,7 @@ server {
     }
 
     ## disable all access to the following directories
-    location ~ ^/(config|tmp|core|lang) {
+    location ~ ^/(config|tmp|core|lang|libs|vendor|plugins|misc|node_modules) {
```

Comment:
Did you check for new folders since the last releases ?

Reply:
non, we were "just" copying the configuration and saw a potential simplification since there are two similar deny rules

Reply:
ok, your Twitter link is 404 Sur ton profil Github

Reply:
good catch, for **reasons** I quit twitter... Fixed.

Reply:
```suggestion
    location ~ ^/(config|tmp|core|lang|libs|tests|vendor|plugins|misc|node_modules) {
```
Here is what is left when I delete all that was included in the current regex
![image](https://github.com/user-attachments/assets/f066db2f-7c5d-4fe3-bd47-440c339c150f)

We should also ensure that console is not accessible from the web. Maybe autoloaders too.
Can you add this ?